examples/request_redraw_threaded.rs

```diff
@@ -0,0 +1,39 @@
+use std::{thread, time};
+
+use winit::{
+    event::{Event, WindowEvent},
+    event_loop::{ControlFlow, EventLoop},
+    window::WindowBuilder,
+};
+
+fn main() {
+    simple_logger::init().unwrap();
+    let event_loop = EventLoop::new();
+
+    let window = WindowBuilder::new()
+        .with_title("A fantastic window!")
+        .build(&event_loop)
+        .unwrap();
+
+    thread::spawn(move || loop {
+        thread::sleep(time::Duration::from_secs(1));
+        window.request_redraw();
```

Comment:
Won't the window get dropped immediately after the call?

Reply:
The window is moved to the closure.

You'll get an error if you also use the window in the event_loop:
```
error[E0382]: use of moved value: `window`
  --> examples\request_redraw_threaded.rs:23:20
   |
13 |     let window = WindowBuilder::new()
   |         ------ move occurs because `window` has type `winit::window::Window`, which does not implement the `Copy` trait
...
18 |     thread::spawn(move || loop {
   |                   ------- value moved into closure here
19 |         thread::sleep(time::Duration::from_secs(1));
20 |         window.request_redraw();
   |         ------ variable moved due to use in closure
...
23 |     event_loop.run(move |event, _, control_flow| {
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ value used here after move
...
35 |                 window.request_redraw();
   |                 ------ use occurs due to use in closure
```

Reply:
Ah, I didn't spot the loop.